Orleans 3.0.0-beta1 Code Generator uses Assemblies for dotnet 2.0 and can't be build in official MS container for dotnet 3.0

The new version Orleans 3.0.0-beta1 still uses assemblies from dotnet 2.0 during code generation. This prevents building the application in the MS docker sdk container `mcr.microsoft.com/dotnet/core/sdk:3.0`.

The generator fails with
```
  It was not possible to find any compatible framework version
  The specified framework 'Microsoft.NETCore.App', version '2.0.0' was not found.
    - The following frameworks were found:
        3.0.0-preview8-28405-07 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

  You can resolve the problem by installing the specified framework and/or SDK.

  The .NET Core frameworks can be found at:
    - https://aka.ms/dotnet-download
/root/.nuget/packages/microsoft.orleans.orleanscodegenerator.build/3.0.0-beta1/build/Microsoft.Orleans.OrleansCodeGenerator.Build.targets(83,5): error MSB3073: The command ""/usr/share/dotnet/dotnet" "/root/.nuget/packages/microsoft.orleans.orleanscodegenerator.build/3.0.0-beta1/build/../tasks/netcoreapp2.0/Orleans.CodeGeneration.Build.dll" "@obj/Debug/netcoreapp3.0/codegen/Eight.Kelvin.GrainInterfaces.orleans.g.args.txt"" exited with code 150. [/root/kelvin-api/src/Eight.Kelvin.GrainInterfaces/Eight.Kelvin.GrainInterfaces.csproj]
```

Installing an older version of dotnet in the container fixes the issue.

--- a/src/BootstrapBuild/Orleans.CodeGeneration.Build/runtimeconfig.template.json
+++ b/src/BootstrapBuild/Orleans.CodeGeneration.Build/runtimeconfig.template.json
@@ -1,0 +1,3 @@
+{
+    "rollForwardOnNoCandidateFx": 2
+}

--- a/src/BootstrapBuild/Orleans.CodeGenerator.MSBuild.Bootstrap/runtimeconfig.template.json
+++ b/src/BootstrapBuild/Orleans.CodeGenerator.MSBuild.Bootstrap/runtimeconfig.template.json
@@ -1,0 +1,3 @@
+{
+    "rollForwardOnNoCandidateFx": 2
+}

--- a/src/Orleans.CodeGeneration.Build/runtimeconfig.template.json
+++ b/src/Orleans.CodeGeneration.Build/runtimeconfig.template.json
@@ -1,0 +1,3 @@
+{
+    "rollForwardOnNoCandidateFx": 2
+}

